units/provisioning_create_host.go

```diff
@@ -178,9 +178,14 @@ func (j *createHostJob) createHost(ctx context.Context) error {
 		"max_attempts": j.MaxAttempts,
 	})
 
-	mgrOpts := cloud.ManagerOpts{
-		Provider: j.host.Provider,
-		Region:   cloud.GetRegion(j.host.Distro),
+	mgrOpts, err := GetManagerOptions(j.host.Distro)
+	if err != nil {
+		grip.Warning(message.WrapError(err, message.Fields{
+			"message": "can't get ManagerOpts for host",
+			"host":    j.host.Id,
+			"job":     j.ID(),
+		}))
```

Comment:
I don't think this logging is needed. errors from the job are already logged.

Reply:
done